### DISCUSSION TITLE
I'm getting errors when starting Orca (Video sink/Javascript). What I'm doing wrong?

### DISCUSSION THREAD

**Discussion by JesConsuegra** [OrcaSlicer/OrcaSlicer#1567] [General]
Just trying Orca on MX-Linux libretto (Based on Debian Bookworm).
Just after launching it I get the following errors:

09:26:04 AM: Could not find a suitable video sink (error 0: Success)
09:26:05 AM: Error running JavaScript: Unsupported result type
09:26:05 AM: Error running JavaScript: Unsupported result type
09:26:05 AM: Error running JavaScript: Unsupported result type

Does anybody know what I'm doing wrong?
Thanks in advance
Jes.

**Comment by voidtrance** [comment]
The JavaScript errors are also present on F38.
